src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/client/dto/DeliusRisk.kt

```diff
@@ -0,0 +1,25 @@
+package uk.gov.justice.digital.hmpps.hmppsallocations.client.dto
+
+import uk.gov.justice.digital.hmpps.hmppsallocations.client.Name
+import java.math.BigInteger
+import java.time.LocalDate
+
+data class DeliusRisk constructor(
+  val name: Name,
+  val crn: String,
+  val activeRegistrations: List<Registrations>,
+  val inactiveRegistrations: List<Registrations>,
+  val ogrs: Ogrs?
+)
+
+data class Registrations constructor(
+  val description: String?,
```

Comment:
I'm surprised a description of a registration would be optional? have you seen this in a call to the API?